### PR TITLE
[Feature] CON 81 - Enable text wrap passages

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to this issue**

https://oat-sa.atlassian.net/browse/CON-81

**Other related PRs**

- https://github.com/oat-sa/extension-tao-itemqti/pull/1580
- https://github.com/oat-sa/extension-tao-mediamanager/pull/240

**Story**

As the author, I want to be able to wrap my text around pictures, so that I can produce passages that look like real-life media content.

Currently, I am only able to anchor images AS a character. In order for the text to be able to flow around it, I will need to be able to anchor it TO a character and specify how text is supposed to wrap around it

Required wrap modes:

- Float off
- Float right
- Float left

**Acceptance Criteria**

- Anchoring an image as a character should still be possible
- Text wrapping should not be available when the image is anchored as a character
- Images maintain position within their paragraph when text is added before or after
- Responsive images are still supported if the image is anchored to a character
- Exported items will contain the relevant CSS to ensure the rendering will be correct in any environment
- Exporting and reimporting an item will maintain the anchoring and wrapping

### CODE REVIEW

**Comment by habemuscode** [comment]
@bugalot I make a change in `manifest.php` because we don't `Updater` any longer more. Can you please validate?

**Comment by habemuscode** [comment]
Comprobation that it's working, in TAO Icons tools in UI

imagen

File: manifest.php [hunk 1]
```diff
@@ -1,7 +1,5 @@
 <?php
 
-use oat\taoDevTools\scripts\update\Updater;
```
Comment (bziondik) [inline manifest.php:3]: I think it should be in manifest

**Review by bziondik — CHANGES_REQUESTED** [review]
Please, check my comment

File: manifest.php (hunk 1 above)
Comment (habemuscode) [reply manifest.php]: ok I put again :D

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
Backend Part is awesome ! 

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**